docs/operate/customize/jobs.md

```diff
@@ -0,0 +1,141 @@
+# How to Create a Custom Job in InvenioRDM
+
+_Added in v13.0.0_
+
+This guide walks developers through implementing a custom job in InvenioRDM using the `invenio-jobs` system. Jobs are asynchronous tasks that can be triggered from the admin UI or REST API. They run using Celery and support logging, argument validation, and result tracking.
+
+---
+
+## Prerequisites
+
+- Python development environment set up with InvenioRDM.
+- Understanding of Celery tasks and entry points.
+- Access to an InvenioRDM instance with administrator rights.
+
+---
+
+## 1. Define Your Celery Task
+
+Create your actual business logic as a Celery task:
```

Comment:
Is there a convention where this task should go in your instance? And same for the Job Class below

Reply:
Not really, it's just an example the users can put them wherever they want, or import from other repos, the important thing is that they simply register them via entrypoints as stated in step 3